crates/tx5-go-pion-sys/build.rs

```diff
@@ -184,25 +184,87 @@ fn go_build() {
             .success(),
         "error running go build",
     );
+}
 
-    use sha2::Digest;
-    let data = std::fs::read(lib_path).expect("failed to read generated lib");
-    let mut hasher = sha2::Sha256::new();
-    hasher.update(data);
-    let hash =
-        base64::encode_config(hasher.finalize(), base64::URL_SAFE_NO_PAD);
-
-    let mut exe_hash = out_dir;
-    exe_hash.push("lib_hash.rs");
-    std::fs::write(
-        exe_hash,
-        format!(
-            r#"
-        const LIB_HASH: &str = "{hash}";
-    "#,
-        ),
-    )
-    .expect("failed to write lib hash");
+fn go_build() {
+    let out_dir = std::env::var("OUT_DIR")
+        .map(std::path::PathBuf::from)
+        .expect("error reading out dir");
+
+    let manifest_path = std::env::var("CARGO_MANIFEST_DIR")
+        .map(std::path::PathBuf::from)
+        .expect("error reading manifest dir");
+
+    let cp = |f: &'static str| {
+        let mut a = manifest_path.clone();
+        a.push(f);
+        let mut b = out_dir.clone();
+        b.push(f);
+        std::fs::copy(a, b).expect("failed to copy go file");
```

Comment:
Technically not only used for Go files, it's a very small thing but something like `Go project file` might be a bit clearer if this were to fail for some reason?